docs/development.md

```diff
@@ -130,6 +130,30 @@ git add glide.yaml glide.lock vendor
 git commit
 ```
 
+#### Updating OpenShift
+
+1. Update the OpenShift version in `scripts/vendor-openshift.sh`.
+
+2. Clear cache
```

Comment:
Just make this more descriptive so you don't have to add the extra sentence below.

`Clear the cache to prevent glide from using previous data`

Reply:
done!